opensrp-giz-malawi/src/test/java/org/smartregister/giz/activity/ReportSummaryActivityTest.java

```diff
@@ -48,7 +51,7 @@ public void setUp() {
     }
 
     @Test
-    public void testOnResume() {
+    public void testOnResumeSetsViewVisible() {
```

Comment:
remove this and add `resume()` on line 49

Reply:
When I move this then it skips the part where it's supposed to actually check on the Visibility

Reply:
Done